frontend/src/Editor/Components/components.js

```diff
@@ -494,6 +495,7 @@ export const componentTypes = [
         format: { value: 'DD/MM/YYYY' },
         enableTime: { value: '{{false}}' },
         enableDate: { value: '{{true}}' },
+        disabledDates: { value: ['1/09/2021'] },
```

Comment:
Let's supply `'{{[]}}'` as the default value. Otherwise the user might assume all dates are allowed.